client/src/Component/views/YourScenario/index.js

```diff
@@ -1,38 +1,90 @@
-          fundraising. The overall aim is to create an environment within which
-          connections and collaborations can take place - the Slipham Connected
-          Commons.
-        </p>
-        <br />
-        <p style={{ fontWeight: 'bold' }}>
-          Your goal is to select the tools that you would use to faciliate
-          creating this Connected Commons Program. You have a limited amount of
-          resources and must choose wisely.
-        </p>
-      </Content>
-      <Footer backLink="/play" nextLink="/persona" reviewScenario={false} />
-    </div>
-  );
-};
+class YourScenario extends Component {
+  state = {
+    slideIndex: 0,
+    // checked: false,
+  };
+
+  afterSlide = slideIndex => {
+    this.setState({ slideIndex });
+  };
+
+  render() {
+    return (
+      <div>
+        <Header headerImg={houses} titleText="Your scenario" />
+        <Content>
+          <p>
+            The Slipham social action centre is launching a Connected Commons
+            programme, aiming to help people explore its heritage, find local
+            opportunities and services, and support new projects for a better
+            Slipham in the future. It will do this by mapping people, projects
+            and groups to find what they can offer, what they need, and how they
+            are connected at present. The Commons won&apos;t be just one
+            physical space. It will be on-the-ground trails and gathering
+            points, maps of doing what, a website and other communication tools,
+            with guides for action. We&apos;ll have small grants to support
+            projects, and help on fundraising. The overall aim is to create an
+            environment within which connections and collaborations can take
+            place - the Slipham Connected Commons.
+          </p>
+          <br />
+          <p style={{ fontWeight: 'bold' }}>
+            Your goal is to select the tools that you would use to faciliate
+            creating this Connected Commons Program. You have a limited amount
+            of resources and must choose wisely.
+          </p>
+        </Content>
+        <Line />
+        <HeadLine>
+          Here are some potential user personas that you can keep in mind while
+          while you choose your methods.
+        </HeadLine>
+        <div>
+          <Carousel
+            wrapAround
+            slideIndex={this.slideIndex}
+            afterSlide={this.afterSlide}
+            enableKeyboardControls
+            renderCenterLeftControls={({ previousSlide }) => (
+              <PrevBtn onClick={previousSlide}>.</PrevBtn>
+            )}
+            renderCenterRightControls={({ nextSlide }) => (
+              <NextBtn onClick={nextSlide}>.</NextBtn>
+            )}
+          >
+            {PersonaData.map(element => {
+              return (
+                <CardWrapper key={element.id}>
+                  <Img heightImg="20rem" src={element.img} alt="PersonImg" />
```

Comment:
You can fix the img height here by just setting height to auto.
![Screen Shot 2019-04-29 at 22 09 25](https://user-images.githubusercontent.com/37771591/56927297-bb03bb00-6acb-11e9-8508-b7e6ad0fd21f.png)

alternatively, if you didn't want to pass props like you did, you could just have styled this as a component. but both work! so nice job! 💯